frontend/src/components/FormInput.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Input, { InputProps } from './Input';
+
+interface FormInputProps extends InputProps {
+  errorMessage?: string;
+}
+
+const FormInput = ({ errorMessage, ...props }: FormInputProps) => {
+  return props.readOnly ? (
```

Comment:
The readonly state of the FormInput should not be an Input component but a <div>